sample-rest-service/src/main/java/com/sl/sample/rest/service/SampleRESTApplication.java

```diff
@@ -2,12 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.annotation.ComponentScan;
 import org.springframework.web.servlet.config.annotation.EnableWebMvc;
 
 @SpringBootApplication
 @EnableWebMvc
-@ComponentScan
```

Comment:
Not needed anymore; underlying Spring Boot enghancements make this annotation redundant.

Reply:
yes good/nice